test/PowerShellEditorServices.Test.E2E/TestsFixture.cs

```diff
@@ -86,7 +85,7 @@ public async Task InitializeAsync()
             _psesProcess = new StdioServerProcess(factory, processStartInfo);
             await _psesProcess.Start();
 
-            await CustomInitializeAsync(factory, _psesProcess);
+            await CustomInitializeAsync(factory, _psesProcess.OutputStream, _psesProcess.InputStream);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5596138)